pysat/tests/test_instrument.py

```diff
@@ -407,6 +438,17 @@ class Dummy: pass
         
         temp = pysat.Instrument(inst_module=Dummy)
 
+class TestBasicsXarray(TestBasics):
+    def setup(self):
+        reload(pysat.instruments.pysat_testing)
+        '''Runs before every method to create a clean testing setup.'''
```

Comment:
single quote mark should be double quotation mark.

Reply:
Fixed